modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/organization/management/v1/OrganizationManagementSuccessTest.java

```diff
@@ -1135,6 +1139,208 @@ public void testDeleteOrganizationsForPagination() {
         Assert.assertTrue(organizations.isEmpty(), "All organizations should be deleted, but the list is not empty.");
     }
 
+    @Test(groups = "organizationMetaAttributesPaginationTests", dependsOnMethods =
+            "testDeleteOrganizationsForPagination")
+    public void createOrganizationsForMetaAttributesPaginationTests() throws JSONException {
+
+        organizations = createOrganizations(1);
+        assertEquals(organizations.size(), 1);
+
+        organizationForMetaAttributes = organizations.get(0);
+        assertNotNull(organizationForMetaAttributes);
```

Comment:
Do we need to create a separate `organizationForMetaAttributes` Map here? Can't we directly use `organizations.get(0)` since we use `organizationForMetaAttributes` in only a few places?

For example:

`String organizationId = organizations.get(0).get("ORGANIZATION_ID");
`

Reply:
Addressed with https://github.com/wso2/product-is/pull/21054/commits/7f95c8efe6686a47acfec90aa0973af6e6e3bcc7